docs/index.rst

```diff
@@ -385,24 +385,32 @@ To discover resources via other parameters
     >>> from hs_restclient import HydroShare, HydroShareAuthBasic
     >>> auth = HydroShareAuthBasic(username='myusername', password='mypassword')
     >>> hs = HydroShare(auth=auth)
-    
+
     >>> # Discover via creator, group, user, owner
-    >>> resources = hs.resources(creator="email or name")
-    >>> resources = hs.resources(user="email or name")
-    >>> resources = hs.resources(owner="email or name")
-    >>> resources = hs.resources(author="email or name")
+    >>> resources = hs.resources(creator="username")
+    >>> resources = hs.resources(user="username")
+    >>> resources = hs.resources(owner="username")
+    >>> resources = hs.resources(author="username")
     >>> resources = hs.resources(group="id or name")
 
+**from_date** allows you to specify the earliest creation date to query for
```

Comment:
I think these dates take specific formats... maybe an example here would nice?

Reply:
I thought they just took datetime objects? Either way I'm happy to include an example

Reply:
Looks like you're correct